libs/sql-ddl/src/common.rs

```diff
@@ -37,3 +40,46 @@ where
         Ok(())
     }
 }
+
+#[derive(Debug, Clone, Copy)]
+pub enum SortOrder {
+    Asc,
+    Desc,
+}
+
+impl Default for SortOrder {
+    fn default() -> Self {
+        Self::Asc
+    }
+}
+
+impl Display for SortOrder {
+    fn fmt(&self, f: &mut Formatter<'_>) -> fmt::Result {
+        f.write_str(self.as_ref())
+    }
+}
+
+impl AsRef<str> for SortOrder {
+    fn as_ref(&self) -> &str {
+        match self {
+            SortOrder::Asc => "ASC",
+            SortOrder::Desc => "DESC",
+        }
+    }
+}
+
+#[derive(Debug, Default)]
+pub struct IndexColumn<'a> {
+    pub name: Cow<'a, str>,
+    pub length: Option<u32>,
+    pub sort_order: Option<SortOrder>,
+}
+
+impl<'a> IndexColumn<'a> {
+    pub fn new(name: &'a str) -> Self {
```

Comment:
The design of `sql-ddl` idea was "only plain structs, no methods". I'm not convinced we still want sql-ddl anymore, so let's not care for now.